nx-X11/lib/include/xtrans/Xtrans.c

```diff
@@ -979,10 +979,11 @@ TRANS(GetMyAddr) (XtransConnInfo ciptr, int *familyp, int *addrlenp,
     if ((*addrp = (Xtransaddr *) malloc (ciptr->addrlen)) == NULL)
     {
        prmsg (1,"GetMyAddr: malloc failed\n");
+       free(addrp);
```

Comment:
This one is not a memleak. The section will only be called if addrp is NULL. So there's no need for a free().

Reply:
ok

Reply:
ohoho, rethink this! we must be careful here and don't confuse addrp and *addrp and **addrp